### ISSUE TITLE
Lost 'page' parameter in _gaCommandSendPageview method

### ISSUE THREAD

**Issue by misiyevich** [PriceRunner/react-ga4#62]
It seems that somebody has lost 'page' parameter in _gaCommandSendPageview method in the place of screenshot (when destcuturing object):
![Ashampoo_Snap_10 января 2023 г _17h26m03s_007_](https://user-images.githubusercontent.com/117369147/211577591-a15a4dc4-0c2b-4034-808b-a956a66d095b.png)

**Comment by nelitow** [comment]
I hope that with the push to GA4, this repo gets some attention.
I'm currently using it and it is working fine, but some better docs would be helpful.

**Comment by jordanburke** [comment]
Agreed, came here after checking out the original version and needed support for GA4. I'd like to see more activity here as well.

**Comment by codler** [comment]
Moved to https://github.com/codler/react-ga4

**Comment by dlong500** [comment]
@codler It would probably be a good idea to archive this repository and add a note about the move in the README.  Thanks for your work!
